skimage/feature/tests/test_util.py

```diff
@@ -86,3 +87,74 @@ def test_plot_matches():
         plot_matches(
             ax, img1, img2, keypoints1, keypoints2, matches, alignment='vertical'
         )
+
+
+@pytest.mark.skipif(not has_mpl, reason="Matplotlib not installed")
+def test_plot_matched_features():
+    from matplotlib import pyplot as plt
+
+    fig, ax = plt.subplots()
+
+    shapes = (
+        ((10, 10), (10, 10)),
+        ((10, 10), (12, 10)),
+        ((10, 10), (10, 12)),
+        ((10, 10), (12, 12)),
+        ((12, 10), (10, 10)),
+        ((10, 12), (10, 10)),
+        ((12, 12), (10, 10)),
+    )
```

Comment:
Any reason not to use `pytest.mark.parametrize("shape0,shape1", ...)` here?

Reply:
Good point; done d98758e1c88e.